specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json

```diff
@@ -2149,6 +2159,20 @@
         "remove"
       ]
     },
+    "UserArtifactSettings": {
+      "type": "object",
+      "properties": {
+        "packageFileName": {
+          "type": "string",
+          "description": "Optional. The name to assign the downloaded package file on the VM. This is limited to 4096 characters. If not specified, the package file will be named the same as the Gallery Application name."
+        },
+        "configFileName": {
+          "type": "string",
+          "description": "Optional. The name to assign the downloaded config file on the VM. This is limited to 4096 characters. If not specified, the config file will be named the Gallery Application name appended with \"_config\"."
```

Comment:
The format \"_config\"." in the description is confusing. Is the \ needed?

Reply:
\ is just the escape the "

Reply:
it will look like "_config" in the text